src/olympia/lib/crypto/tasks.py

```diff
@@ -149,30 +144,24 @@ def bump_addon_version(old_version):
 
     try:
         # Copy the original file to a new FileUpload.
-        task_user = get_task_user()
-        # last login ip should already be set in the database even on the
-        # task user, but in case it's not, like in tests/local envs, set it
-        # on the instance, forcing it to be localhost, that should be
-        # enough for our needs.
-        task_user.last_login_ip = '127.0.0.1'
         original_author = addon.authors.first()
         upload = FileUpload.objects.create(
             addon=addon,
-            version=bumped_version_number,
-            user=task_user,
-            channel=amo.CHANNEL_LISTED,
+            version=new_version_number,
+            user=user,
```

Comment:
Is it a requirement that the user is an actual addon author? Why not leave it as task_user?

 I could almost see some value in distinguishing between "machine" uploaded files and "user" uploaded files.. even if the user is "clicking the button".
 
 If we create the FileUpload entity with the actual user, is there any way (currently) to distinguish between files created via this duplication or normally uploaded files?
 
 I don't have a specific use case in mind for that information but it is at least worth considering? If something were to be wrong with these created files after creation, being able to query for them could be useful to fix DB state. Or if we want to clean up machine created files for some other reason?